.evergreen/print-compass-env.sh

```diff
@@ -0,0 +1,6 @@
+#! /usr/bin/env bash
+
+export BASHPATH=$PATH
+export OSTYPE=$OSTYPE
```

Comment:
why is this needed?

Reply:
Commented elsewhere but it is a variable only bash sets internally and as far as I can tell it isn't exported. This is how I finally found it https://github.com/dylanaraps/neofetch/issues/433

Reply:
if is available here is available to the subprocess, couldn't you read it from the `.js` file?

Reply:
No and the reason is because it isn't exported. In a terminal type `env | grep OSTYPE` and you'll see it is empty. Then `echo $OSTYPE` and it prints `darwin20.0`